### PR TITLE
Doc for analytic functions

### PR DESCRIPTION
**Pull request by CorvusCorrax**
Add docs for denseRank(), rank() and rowNumber().
Also fix broken del() link in sidebar (https://github.com/knex/documentation/issues/3)

### CODE REVIEW

**Comment by elhigu** [comment]
Built files should not be in PR... those are updated usually only between new releases.

**Comment by kibertoad** [comment]
@elhigu Is this PR still useful? I can create an updated version without built artifacts.

File: components/Sidebar.jsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ export default class Sidebar extends Component {
           <li>– <a href="#Builder-insert">insert</a></li>
           <li>– <a href="#Builder-returning">returning</a></li>
           <li>– <a href="#Builder-update">update</a></li>
-          <li>– <a href="#Builder-del">del / delete</a></li>
+          <li>– <a href="#Builder-del / delete">del / delete</a></li>
```
Comment (elhigu) [inline components/Sidebar.jsx]: Is this href link correct?

**Comment by elhigu** [comment]
@kibertoad Looks like @tgriesser didn't approve the related PR yet so no need to update docs yet either.

**Pull request merged**